packs/data/equipment-effects.db/effect-earplugs-pfs-guide.json

```
@@ -0,0 +1,59 @@
+{
+    "_id": "NE7Fm5YnUhD4ySX3",
+    "img": "systems/pf2e/icons/equipment/adventuring-gear/earplugs.webp",
+    "name": "Effect: Earplugs (PFS Guide)",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.iYiA6CMR1N2E2Ny2]{Earplugs (PFS Guide)}</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 0
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "label": "Earplugs (vs. auditory)",
+                "predicate": [
+                    "auditory"
+                ],
+                "selector": "saving-throw",
+                "type": "item",
+                "value": 1
+            },
+            {
+                "key": "FlatModifier",
+                "label": "Earplugs (hearing)",
+                "predicate": [
+                    "hearing"
```

```suggestion
                    "auditory"
```
Should this be `auditory` as well? If you were making an inline perception check you would likely use the standard trait name.